ios-base/DroidKaigi 2020/SideBar/Views/Cells/SidebarCell.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+final class SidebarCell: UITableViewCell {
+    let notSelectedColor = UIColor(hex: "A8A8A8")
+
+    @IBOutlet weak var iconImageView: UIImageView! {
+        didSet {
+            iconImageView.image?.withRenderingMode(.alwaysTemplate)
```

Comment:
Maybe this is `iconImageView.image = iconImageView.image?.withRenderingMode(.alwaysTemplate)`.